### ISSUE TITLE
HelloPanorama code is not building 

### ISSUE THREAD

**Issue by GoogleCodeExporter** [niltsh/panoramagl#9]
```
What steps will reproduce the problem?
1. Issues with GLUquadric. Error : Unknown type name GLUquadric.
2.
3.

What is the expected output? What do you see instead?
Build the code. Errors

What version of the product are you using? On what operating system?
Xcode 4.2, Simulator 5.0

Please provide any additional information below.


```

Original issue reported on code.google.com by `surabhia...@gmail.com` on 10 Jan 2012 at 9:28

**Comment by GoogleCodeExporter** [comment]
```
Solved it using the comments here:
http://code.google.com/p/panoramagl/issues/detail?id=2
```

Original comment by `bur...@gmail.com` on 23 Jan 2012 at 8:58

**Comment by GoogleCodeExporter** [comment]
```
This problem was corrected in 0.1 version
```

Original comment by `javbae...@gmail.com` on 5 Jul 2012 at 3:51
- Changed state: **Fixed**
